datastore/graphers/nmdb-graph-network/nmdb-graph-network.cpp

```diff
@@ -671,10 +675,15 @@ class Tool : public nmdt::AbstractGraphTool
     {
       std::string label = "\" + <<TABLE border=\"0\" cellborder=\"0\"><TR>";
 
-      label += getUseIconString(typeName);
-
-      label += "<td><font point-size=\"10\">" + name + "<br/>";
+      if (useIcons) {
+        label += getUseIconString(typeName);
+        label += "<td><font point-size=\"10\">" + name + "<br/>";
+      }
 
+      if (useIconFolder) {
+        label += getUseIconsCustomPath(typeName);
+        label += "<td><font point-size=\"10\">" + name + "<br/>";
+      }
```

Comment:
You should not need to make any edits to this code section.  The `getUseIconString()` function is what (currently) fills in the path for the icon folder, among other things.